### ISSUE TITLE
Actors subsection hard to follow

### ISSUE THREAD

**Issue by wweatherill-answer** [nhsconnect/FHIR-NRLS-API#7]
The Actors subsection of the Concepts page does not clearly highlight all of the Actors. Consider moving them into a table with the headings Actor | Role.

**Comment by wilkinsm** [comment]
Additional comments and feedback on this have been included [here](https://github.com/nhsconnect/FHIR-NRLS-API/issues/26)

**Comment by wweatherill-answer** [comment]
Section now updated with improved formatting to aid readability.
